src/qmapshack/dem/CDemPropSetup.cpp

```diff
@@ -121,6 +127,18 @@ void CDemPropSetup::slotPropertiesChanged()
         slopeSpins[i]->setValue(demfile->getCurrentSlopeStepTable()[i]);
     }
 
+    checkElevationLimit->setChecked(demfile->doElevationLimit());
+    spinBoxElevationLimit->setValue(demfile->getElevationLimit());
+
+    qreal meter = 0;
+    qreal val = 0;
+    QString elevationUnit;
+    IUnit::self().meter2elevation(meter, val, elevationUnit);
```

Comment:
Not really what I meant but let's leave it as it is. I opened a follow up ticket to cleanup the mess.

https://github.com/Maproom/qmapshack/issues/64